package.json

```diff
@@ -37,48 +38,48 @@
     "debug": "^4.3.2",
     "detect-indent": "^6.1.0",
     "detect-newline": "^3.1.0",
-    "execa": "^5.1.1",
+    "execa": "^6.1.0",
     "get-stream": "^6.0.1",
     "git-log-parser": "^1.2.0",
-    "globby": "^11.0.4",
-    "lodash": "^4.17.21",
+    "globby": "^13.0.0",
+    "lodash-es": "^4.17.21",
     "meow": "^9.0.0",
     "promise-events": "^0.2.4",
-    "semantic-release": "npm:@killianhmyd/semantic-release@0.0.0-notes-fix.7",
+    "semantic-release": "npm:@killianhmyd/semantic-release@0.0.0-notes-fix.9",
     "semver": "^7.3.5",
     "signale": "^1.4.0",
     "stream-buffers": "^3.0.2"
   },
   "devDependencies": {
-    "@commitlint/config-conventional": "17.4.0",
+    "@commitlint/config-conventional": "17.3.0",
     "@schemastore/package": "0.0.6",
     "@types/cosmiconfig": "6.0.0",
     "@types/debug": "4.1.7",
     "@types/execa": "2.0.0",
-    "@types/jest": "29.2.5",
-    "@types/lodash": "4.14.191",
-    "@types/node": "18.11.18",
+    "@types/jest": "29.2.4",
+    "@types/lodash-es": "4.17.6",
+    "@types/node": "18.11.16",
     "@types/semantic-release": "17.2.4",
     "@types/semver": "7.3.13",
     "@types/signale": "1.4.4",
     "@types/stream-buffers": "3.0.4",
-    "@typescript-eslint/eslint-plugin": "5.48.1",
-    "@typescript-eslint/parser": "5.48.1",
-    "commitlint": "17.4.1",
+    "@typescript-eslint/eslint-plugin": "5.46.1",
+    "@typescript-eslint/parser": "5.46.1",
+    "commitlint": "17.3.0",
     "coveralls": "3.1.1",
-    "eslint": "8.31.0",
-    "eslint-config-prettier": "8.6.0",
+    "eslint": "8.30.0",
+    "eslint-config-prettier": "8.5.0",
     "eslint-config-standard-with-typescript": "24.0.0",
     "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jest": "27.2.1",
+    "eslint-plugin-jest": "27.1.7",
     "eslint-plugin-n": "15.6.0",
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-promise": "6.1.1",
     "file-url": "3.0.0",
-    "husky": "8.0.3",
+    "husky": "8.0.2",
     "jest": "29.3.1",
     "lint-staged": "13.1.0",
-    "prettier": "2.8.2",
+    "prettier": "2.8.1",
```

Comment:
not sure to understand why you downgraded some dependencies here 🤔

Reply:
No idea 🤔 Maybe I just got an issue with my rebasing or else.... I'll fix it